server/scsynth/SC_MiscCmds.cpp

```diff
@@ -1326,6 +1327,29 @@ SCErr meth_dumpOSC(World *inWorld, int inSize, char *inData, ReplyAddress *inRep
 	return kSCErr_None;
 }
 
+SCErr meth_version(World *inWorld, int inSize, char *inData, ReplyAddress *inReply);
+SCErr meth_version(World *inWorld, int inSize, char *inData, ReplyAddress* inReply)
+{
+	sc_msg_iter msg(inSize, inData);
+
+	small_scpacket packet;
+	packet.adds("/version.reply");
+	packet.maketags(5);
+	packet.addtag(',');
+	packet.addtag('s');
+	packet.adds("scsynth");
+	packet.addtag('i');
+	packet.addi(SC_VersionMajor);
+	packet.addtag('i');
+	packet.addi(SC_VersionMinor);
+	packet.addtag('s');
+	packet.adds(SC_VersionPatch);
+
```

Comment:
SC_CommitHash is enough to tell us exactly what code we're running (unless the user made mods of course :smile: ). Then again, if this would be useful to someone there's no real reason not to add it